tests/lib/nm/device_test.py

```diff
@@ -41,10 +41,11 @@ def mainloop_mock():
 
 def test_activate(client_mock, mainloop_mock):
     dev = 'foodev'
-    nm.device.activate(dev)
+    mainloop_mock.push_action = lambda func, dev, fetch: func(dev, fetch)
 
-    mainloop_mock.push_action.assert_called_once_with(
-        client_mock.activate_connection_async,
+    nm.device.activate(dev, fetch_remote_connection_func=None)
```

Comment:
The fetch function is by default None, why specify it explicitly here? It seems that there is a test missing in case it is not None.

Reply:
Done

Reply:
Regarding the missing test: It is covered by the nm.vlan test in the integration suite.
I will start taking out these unit nm tests in future patches.